### PR TITLE
docs: add nb for image tiling workflow

### PR DESCRIPTION
**Pull request by cpaniaguam**
- Illustrative notebook
- Update function docstrings

### CODE REVIEW

File: notebooks/preprocessing-workflow/tiling.ipynb [hunk 1]
```diff
@@ -0,0 +1,229 @@
+{
```
Comment (tdivoll) [inline notebooks/preprocessing-workflow/tiling.ipynb:1]: <p>Maybe just start with a sentence or two describing what the intent of the notebook is so novice users can decide if this notebook provides the example they are looking for? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WilhelmusLab/IceFloeTracker.jl/pull/434/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by tdivoll — APPROVED** [review]

File: notebooks/preprocessing-workflow/tiling.ipynb (hunk 1 above)
Comment (cpaniaguam) [reply notebooks/preprocessing-workflow/tiling.ipynb:1]: <p>What about a markdown cell at the top with a title and a sentence fulling that purpose?</p><p> </p><p>I think adding a README to the folder discussing what the notebooks are about might be useful also.</p>

Comment (danielmwatkins) [reply notebooks/preprocessing-workflow/tiling.ipynb:1]: Yes, I think a markdown cell and an informative title would do the trick.

File: notebooks/preprocessing-workflow/tiling.ipynb [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    "#### Load the image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "img = load(\"bering_strait/truecolor/20200504.aqua.truecolor.250m.tiff\")"
```
Comment (danielmwatkins) [inline notebooks/preprocessing-workflow/tiling.ipynb:52]: Should this point to the test images folder?

Comment (cpaniaguam) [reply notebooks/preprocessing-workflow/tiling.ipynb:52]: Oh, I guess those images are not in this branch yet. I'll use one from the test folder, thanks!

**Review by danielmwatkins — APPROVED** [review]
Updated notebook looks good to me!

**Pull request merged**